tests/integration/cartography/intel/snipeit/test_snipeit_assets.py

```diff
@@ -0,0 +1,183 @@
+import logging
+
+import cartography.intel.snipeit
+import tests.data.snipeit.assets
+import tests.data.snipeit.tenants
+import tests.data.snipeit.users
+from tests.integration.util import check_nodes
+from tests.integration.util import check_rels
+
+logger = logging.getLogger(__name__)
+
+
+def test_load_snipeit_assets_relationship(neo4j_session):
+    # Arrange
+    TEST_UPDATE_TAG = 1234
+    TEST_snipeit_TENANT_ID = tests.data.snipeit.tenants.TENANTS["company_a"]["id"]
+    common_job_parameters = {
+        "UPDATE_TAG": TEST_UPDATE_TAG,
+        "TENANT_ID": TEST_snipeit_TENANT_ID,
+    }
+
+    # Load test users for the relationship
+    data = tests.data.snipeit.users.USERS['company_a']
+    cartography.intel.snipeit.user.load_users(
+        neo4j_session,
+        common_job_parameters,
+        data,
+    )
+
+    data = tests.data.snipeit.assets.ASSETS['company_a']
+
+    # Act
+    cartography.intel.snipeit.asset.load_assets(
+        neo4j_session,
+        common_job_parameters,
+        data,
+    )
+
+    # Assert
+
+    # Make sure the expected Tenant is created
+    expected_nodes = {
+        ('Company A',),
+    }
+    check_nodes(
+        neo4j_session,
+        'snipeitTenant',
+        ['id'],
+    )
+
+    # Make sure the expected assets are created
+    expected_nodes = {
+        (1373, "C02ZJ48XXXXX"),
+        (1372, "72ec94a8-b6dc-37f1-b2a9-0907806e8db7"),
+    }
+    assert check_nodes(
```

Comment:
Nit: could rewrite as 

```python
assert check_nodes(
        neo4j_session,
        "SnipeitAsset",
        ["id", "serial"],
    ) == {
        (1373, "C02ZJ48XXXXX"),
        (1372, "72ec94a8-b6dc-37f1-b2a9-0907806e8db7"),
}
```

to save a couple lines

Reply:
I will leave these for now (following same pattern for all tests) and will keep in mind for future tests.